index.html

```diff
@@ -518,11 +518,12 @@ <h2>Algorithms</h2>
       </p>
 
       <p>
-When the RDF Dataset Canonicalization Algorithm [[RDF-CANON]] is used with ECDSA
-algorithms, the cryptographic hashing function that is passed to the algorithm
-MUST be determined by the size of the associated public key. For P-256 keys,
-SHA-2 with 256 bits of output is utilized. For P-384 keys, SHA-2 with 384-bits
-of output is utilized.
+When the RDF Dataset Canonicalization Algorithm (RDFC-1.0) [[RDF-CANON]] is used with ECDSA
+algorithms, the cryptographic hashing function used by RDFC-1.0 MUST depend on the size of 
```

Comment:
This first MUST seems a bit awkward now that the following sentences have the more clarifying and more obviously testable MUSTs in them. Can we do this?:

```suggestion
algorithms, the cryptographic hashing function used by RDFC-1.0 depends on the size of 
```

Approving either way.

Reply:
@dlongley, can you re-review? I have incorporated @TallTed's change proposal, and it may cover your comment as well...

Reply:
It does read a bit better now, but I still think the first MUST is unnecessary because it isn't specific -- it's the following two sentences that make the specific asks of implementations. I'd still change it from "MUST depend" to just "depends".

Reply:
I have no idea what happened, but it seems as @TallTed's change proposals, that I thought I accepted, did not make it into the branch (I may have pushed the "resolve" button too quickly.) I re-did his editings manually.

To avoid misunderstandings, this is the text now:

> When the RDF Dataset Canonicalization Algorithm (RDFC-1.0) [[RDF-CANON]] is used
with ECDSA algorithms, the cryptographic hashing function used by RDFC-1.0 MUST
be chosen based on the size of the associated public key. For P-256 keys, the
default hashing function, SHA-2 with 256 bits of output, MUST be used. For P-384
keys, SHA-2 with 384-bits of output MUST be used, specified via the RDFC-1.0
<a data-cite="RDF-CANON#dfn-hash-algorithm">implementation-specific parameter</a>.

In this text I believe all _MUST_-s are justified. @dlongley, can you look at it again?

(Sorry for my clumsiness...)